src/main/java/fr/lunatech/timekeeper/timeutils/CalendarFR2021.java

```diff
@@ -0,0 +1,76 @@
+package fr.lunatech.timekeeper.timeutils;
+
```

Comment:
Did you clone the code from calendar fr2020? to avoid that I think you should start to work on the API to get off days

Reply:
You are right. It is a subject already in view and will be discussed in the next tech meeting.